{"changed":true,"filter":false,"title":"simple.py","tooltip":"/my_package/simple.py","value":"#! /usr/bin/env python\n\nimport rospy\n\nrospy.init_node('ObiWan')\nprint \"Help me Obi-Wan Kenobi, you're my only hope\"","undoManager":{"mark":-2,"position":0,"stack":[[{"start":{"row":0,"column":0},"end":{"row":5,"column":51},"action":"insert","lines":["#! /usr/bin/env python","","import rospy","","rospy.init_node('ObiWan')","print \"Help me Obi-Wan Kenobi, you're my only hope\""],"id":1}]]},"ace":{"folds":[],"scrolltop":0,"scrollleft":0,"selection":{"start":{"row":5,"column":51},"end":{"row":5,"column":51},"isBackwards":false},"options":{"guessTabSize":true,"useWrapMode":false,"wrapToView":true},"firstLineState":0},"timestamp":1538450660762}